index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Montserrat Cruz | Portfolio</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header class="header-nav">
+    <a href="#" id="mylogo">My Logo</a>
+    <nav class="navMenu">
+  
+        
+        <div id="menu">
+            <i class="bar"></i>
+            <i class="bar"></i>
+            <i class="bar"></i>
+        </div>
+         
+        
+      </nav>
+
```

Comment:
According to the changes required by the previous reviewer, Kindly, I suggest you remove those empty lines from your code to make it look more professional and well-indented. 😄